classes/Provider/ResetEmailerServiceProvider.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace OpenCFP\Provider;
+
+use OpenCFP\Domain\Services\ResetEmailer;
+use Silex\Application;
+use Silex\ServiceProviderInterface;
+
+class ResetEmailerServiceProvider implements ServiceProviderInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function register(Application $app)
```

Comment:
This is a little sneaky... we're calling `$app->config()` below, which is from `\OpenCFP\Application` not `\Silex\Application` as in the `use`... but the interface requires `\Silex\Application` and PHP doesn't complain, so... ?